src/content/docs/de/guides/troubleshooting.mdx

```diff
@@ -0,0 +1,208 @@
+
+```shell ins="react react-dom"
+# Beispiel: Integrationen und Frameworks gemeinsam installieren
+npm install @astrojs/react react react-dom
+```
+Im [Astro-Integrationsleitfaden](/de/guides/integrations-guide/) findest du Anweisungen zum Hinzufügen von Framework-Renderern, CSS-Tools und anderen Paketen für Astro.
+
+### `Astro.glob()` - no matches found
+
+Wenn du `Astro.glob()` verwendest, um Dateien zu importieren, achte darauf, dass du die richtige Glob-Syntax verwendest, die alle benötigten Dateien abdeckt.
+
+#### Dateipfade
+
+Verwende zum Beispiel `../components/**/*.js` in `src/pages/index.astro`, um die beiden folgenden Dateien zu importieren:
+- `src/components/MyComponent.js`
+- `src/components/includes/MyOtherComponent.js`
+
+#### Unterstützte Werte
+
+ `astro.glob()` unterstützt keine dynamischen Variablen und keine Zeichenketten-Interpolation.
+
+Dies ist kein Fehler in Astro. Das liegt an einer Einschränkung der [Vite-Funktion `import.meta.glob()`](https://vitejs.dev/guide/features.html#glob-import), die nur statische Zeichenketten-Literale unterstützt.
+
+Eine gängige Lösung ist es, stattdessen eine größere Gruppe von Dateien zu importieren, die alle benötigten Dateien enthält, und diese dann mit `Astro.glob()` zu filtern:
+
+```astro {6-7}
+---
+// src/components/featured.astro
+const { postSlug } = Astro.props;
+const pathToMyFeaturedPost = `src/pages/blog/${postSlug}.md`;
+
+const posts = await Astro.glob('../pages/blog/*.md');
+const myFeaturedPost = posts.find(post => post.file.includes(pathToMyFeaturedPost));
+---
+
+<p>
+  Wirf einen Blick auf meinen Lieblingspost, <a href={myFeaturedPost.url}>{myFeaturedPost.frontmatter.title}</a>!
+</p>
+```
+
+### Astro mit Yarn 2+ (Berry) verwenden
+
+Yarn 2+, auch bekannt als Berry, verwendet eine Technik namens [Plug'n'Play (PnP)](https://yarnpkg.com/features/pnp), um Node-Module zu speichern und zu verwalten, die [Probleme](https://github.com/withastro/astro/issues/3450) beim Initialisieren eines neuen Astro-Projekts mit `create astro` oder bei der Arbeit mit Astro verursachen kann. Dies kann man umgehen, indem man die [Eigenschaft `nodeLinker`](https://yarnpkg.com/configuration/yarnrc#nodeLinker) in `.yarnrc.yml` auf `node-modules` setzt:
+
+```yaml title=".yarnrc.yml"
+nodeLinker: "node-modules"
+```
+
+### Verwendung von `<head>` in einer Komponente
+
+In Astro funktioniert der `<head>`-Tag wie jeder andere HTML-Tag: Er wird nicht an den Anfang der Seite verschoben oder mit dem vorhandenen `<head>` zusammengeführt. Deshalb solltest du in der Regel nur ein einziges `<head>`-Tag auf einer Seite verwenden. Wir empfehlen, diesen einzelnen `<head>` und seinen Inhalt in eine [Layout-Komponente](/de/core-concepts/layouts/) zu schreiben.
+
+### Ein unerwartetes `<script>` oder `<style>` ist enthalten
```

Comment:
Ich wuerde hier die schreibweise aendern und in den Titel aufnehmen, dass es sich um die Tags handelt.

```suggestion
### Ein unerwarteter `<script>`- oder `<style>`-Tag  ist enthalten
```

Reply:
👍